Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/workspace/WorkspaceCCDAO.java

```diff
@@ -97,11 +96,8 @@ public String addWorkspaceCC(String workspaceId, String CCId, String userId) thr
 		}
 		catch(Exception e)
 		{
-			errMsg = "Oops! Error from DB";
-			logger.error("addWorkspaceCC method :",e);
         	throw new QuadrigaStorageException();

```

Comment:
Please catch the specific exception being thrown and initialize the new exception with the one being thrown. otherwise we loose the stacktrace.